src/observer_cli.erl

```diff
@@ -69,52 +76,56 @@ manager(ChildPid, Opts) ->
         _ -> manager(ChildPid, Opts)
     end.
 
-render_worker(#home{} = Home) ->
+render_worker(TerminalRow, #home{} = Home) ->
     ?output(?CLEAR),
     StableInfo = get_stable_system_info(),
-    redraw(running, StableInfo, erlang:make_ref(), 0, Home).
+    redraw(running, StableInfo, erlang:make_ref(), 0, Home, TerminalRow).
 
 %% pause status waiting to be resume
-redraw(pause, StableInfo, LastTimeRef, _, #home{func = Func, type = Type} = Home) ->
+redraw(pause, StableInfo, LastTimeRef, _, #home{func = Func, type = Type} = Home, TerminalRow) ->
     notify_pause_status(),
     erlang:cancel_timer(LastTimeRef),
     receive
         quit -> quit;
         pause_or_resume ->
             ?output(?CLEAR),
-            redraw(running, StableInfo, LastTimeRef, ?FAST_COLLECT_INTERVAL, Home);
+            redraw(running, StableInfo, LastTimeRef, ?FAST_COLLECT_INTERVAL, Home, TerminalRow);
         {Func, Type} ->
-            redraw(running, StableInfo, LastTimeRef, ?FAST_COLLECT_INTERVAL, Home)
+            redraw(running, StableInfo, LastTimeRef, ?FAST_COLLECT_INTERVAL, Home, TerminalRow)
     end;
 %% running status
 redraw(running, StableInfo, LastTimeRef, NodeStatsCostTime, #home{tid = Tid,
-    interval = Interval, func = Func, type = Type, cur_pos = RankPos} = Home) ->
+    interval = Interval, func = Func, type = Type, cur_pos = RankPos} = Home, TerminalRow0) ->
+    {ok, TerminalRow} =
```

Comment:
4 places use the same code,
Maybe write a new function will better, such as `observer_cli_lib:to_row/1)`